iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NetworkImpl.java

```diff
@@ -727,6 +765,12 @@ protected SynchronousComponentImpl createComponent(int num, int size) {
         private final SynchronousComponentsManager synchronousComponentsManager
                 = new SynchronousComponentsManager(NetworkImpl.this);
 
+        private final BusCache busViewCache = new BusCache(() -> getVoltageLevelStream().flatMap(vl -> vl.getBusView().getBusStream()));
+
+        private final BusCache busBreakerViewCache = new BusCache(() ->
+            getVoltageLevelStream().filter(vl -> vl.getTopologyKind() != TopologyKind.BUS_BREAKER)
```

Comment:
The filtering is a bit confusing at first! Could you add a short comment here to say that you filter out the bus breaker ones because of `NetworkIndex`?